From: =?utf-8?q?Laurent=20L=C3=A9onard?= <laurent@open-minds.org>
Date: Sat, 25 Jul 2009 21:49:06 +0200
Subject: [PATCH] Change virt-what-cpuid-helper path to /usr/lib/virt-what.

---
 virt-what.in |    2 +-
 1 files changed, 1 insertions(+), 1 deletions(-)

diff --git a/virt-what.in b/virt-what.in
index cc61aae..d0ac095 100644
--- a/virt-what.in
+++ b/virt-what.in
@@ -90,7 +90,7 @@ fi
 # To tell if it is Xen and KVM HVM (fully virtualized) we can use this
 # helper C program.
 
-cpuid=`virt-what-cpuid-helper`
+cpuid=`/usr/lib/virt-what/virt-what-cpuid-helper`
 
 # Check for Xen.
 
-- 
